.github/ISSUE_TEMPLATE/nvt-min.lua

```diff
@@ -11,6 +11,10 @@ local function load_plugins()
       "wbthomason/packer.nvim",
       "nvim-tree/nvim-tree.lua",
       "nvim-tree/nvim-web-devicons",
+
+      -- UNCOMMENT for diagnostics issues
+      -- "neovim/nvim-lspconfig",
```

Comment:
Shouldn't we request native lsp start function call over additional plugin?

Reply:
Yes please!

Do you know how to get that going?

Reply:
I guess we can just direct to `:h vim.lsp.start` which has an example right there.

Reply:
Well... I was overthinking that; I didn't realise it was that simple.

Please test drive the new commented out template.